.env

```diff
@@ -0,0 +1,3 @@
+REACT_APP_SERVER_URL=https://mike-n-jacobs-can-of-books.herokuapp.com
+PORT:3001
+
```

Comment:
- `.env` was committed so we just need to ensure that it is properly added to `.gitignore` 👍